plugins/view-webhook.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: view-webhook
+spec:
+  version: v0.0.3
+  homepage: https://github.com/Trendyol/kubectl-view-webhook
+  shortDescription: Visualize your webhook configurations in Kubernetes.
+  description: |
+    👀 This projects aims to visualize critical parts of the admission webhook configuration resource
```

Comment:
- Let's avoid emojis. Many terminals still can't render them.
- Please line wrap around 80 chars.
- `This projects aims to` doesn't add anything, remove.

Reply:
Thanks to your valuable comments, done btw. 🙏